content/en/dynamic_instrumentation/how-it-works.md

```diff
@@ -7,50 +7,49 @@ private: true
 
 ## Overview
 
-Dynamic instrumentation allows you to add probes to your production systems at any point in your code, even in third-party libraries. Dynamic Instrumentation has low overhead and is guaranteed to have no side effects on your system.
+Dynamic instrumentation allows you to add probes to your production systems at any location in your application's code, including third-party libraries. Dynamic Instrumentation has low overhead and is guaranteed to have no side effects on your system.
 
 ## Probe types
 
-You can create *snapshot probes* and *metric probes*.
+You can create *log probes* and *metric probes*.
 
-### Snapshot probes
+### Log probes
 
-Snapshot probes capture the context of a method or line, including:
+Log probes are enabled by default on all service instances that match the desired environment and version. They are rate limited to 5000 logs/s.
```

Comment:
I would also add a line with example on to write a template message: 
"User {user.id} completed transaction with {count(products)} products"

And also mention that conditions can be used to create logs on specific cases an maybe give an example of condition.

Reply:
:+1: Good call, added this example and a paragraph about conditions on this page. I didn't put an example for a condition in, we have one in the screenshot on the main page.